conseil-api/src/main/resources/developer.conf

```diff
@@ -1,3 +1,4 @@
 # Configuration file for local development.
+include required(classpath("application.conf"))
```

Comment:
are you sure that you need `"application.conf"` and not simply `"application"` ?

Reply:
Both works. I don't need it, but since this file is managed by "us", I don't see the reason to make this change. Extension is not going to change either way, right?